tools/autograd/gen_variable_type.py

```diff
@@ -317,6 +317,29 @@
 ${return_type} ${api_name}(${type_method_formals}); // {"schema": "${schema_string}", "compound": "${compound}"}
 """)
 
+# ProfiledType templates
+PROFILE_DISPATCH_UNBOXED = CodeTemplate("""\
+auto op = c10::Dispatcher::singleton().findSchema({"aten::${operator_name}", "${overload_name}"});
```

Comment:
This should probably be `static auto op` so you don't have to do a find schema every time you run this